sw/applications/example_dma/main.c

```diff
@@ -18,25 +18,31 @@
 #define TEST_ADDRESS_MODE_EXTERNAL_DEVICE
 
 #define TEST_DATA_SIZE      16
-#define TEST_DATA_LARGE     4096
+#define TEST_DATA_LARGE     1024
```

Comment:
you can put back 4096 if you want, or does it compile with 2 banks like this?

in case it does not, please guard the application to run only with MEMBANKS > 2

otherwise warning and report 0

Reply:
nice